src/frontend/screens/Settings/Settings.js

```diff
@@ -126,6 +131,17 @@ const Settings = () => {
           secondary={<FormattedMessage {...m.projectConfigDesc} />}
         ></ListItemText>
       </ListItem>
+      <ListItem
+        onPress={() => navigate("MapSettings")}
+        testID="settingsMapSettingsButton"
+      >
+        <ListItemIcon iconName="map" />
+        <ListItemText
+          primary={<FormattedMessage {...m.mapSettings} />}
+          secondary="---------"
```

Comment:
Can we either omit or add something more meaningful here?

Reply:
This was in the mockups

Reply:
ok i would ask to confirm that's what we want, but not a blocker for the PR

Reply:
Lets just merge this so  we can get the Release, and we can fix it if Afterwards if sabella decides to put something else